src/library_pthread.js

```diff
@@ -212,7 +212,7 @@ var LibraryPThread = {
       // Call thread init functions (these are the emscripten_tls_init for each
       // module loaded.
       for (var i in PThread.tlsInitFunctions) {
-        PThread.tlsInitFunctions[i]();
+        if(Pthread.tlsInitFunctions.hasOwnProperty(i)) PThread.tlsInitFunctions[i]();
```

Comment:
Sorry for the back and forth but can we put a space after keywords like `if`.

Reply:
Oh, yeah, sorry, was incautious here, fell back into my usual code style.